.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,11 @@
-## What kind of a pull request is this?
-
-- QA / Code Review
-<!-- Add other options if appropriate -->
-
+## Description of changes
+<!-- It is often obvious what changed by looking at the code, so it is more helpful to say _why_ it should be changed -->
+<!-- If the Pull Request is not ready to be merged, please use a draft pull request -->
 
 ## Code Checklist
 
-- [ ] The code follows dotkom code style 
-- [ ] The code passes the defined tests
 - [ ] I have added tests for the code I added
 - [ ] I have provided documentation for the code I added
```

Comment:
Keep this for now. But we should probably have a talk about documentation again. I can't remember ever writing documentation for a OW change, for good and for bad.